packages/edit-site/src/components/style-book/index.js

```diff
@@ -79,7 +79,7 @@ function getExamples() {
 	return [ headingsExample, ...otherExamples ];
 }
 
-function StyleBook( { isSelected, onSelect, onClose } ) {
+function StyleBook( { selectedBlockPath, onSelect, onClose } ) {
```

Comment:
I don't think `StyleBook` should be concerned with what a "path" is. `StyleBook` can be rendered in any context independently of whether or not a `Navigator` is used. I suggest instead adding a `selectedCategory: string` or `isCategorySelected: ( category: string ) => boolean` prop to `StyleBook` and moving the path logic up one level.